web/src/lib/UpdateQueue.ts

```diff
@@ -48,11 +48,18 @@ export class UpdateQueueFactory implements UpdateQueue {
     return this;
   }
 
-  queueFormationData(formationData: Partial<FormationData>): UpdateQueue {
+  queueFormationData(
+    formationData: Partial<FormationData>,
+    formationFormData?: Partial<FormationFormData>
```

Comment:
I'm thinking I would rather write  function `queueFormationFormData` than add a second parameter to this one. 

Here's my reasons why:
This breaks the pattern of these queue functions, which so far are very consistent. they are named "queueType" and they take a parameter that is of type Partial<Type> and then they spread everything else in userdata while updating Type with the params passed in. It's a little weird to expect people to break with this pattern and have something take two parameters. It also opens us up to a whole world of muddying up these functions to take arbitrary variables. By this model, we might as well have any function take any number of sub-objects and then we'd wind up with the unreadableness of the original `update` function. I'd much rather keep them very explicit and declarative: it queues ONE subtype object and inserts it into the userData.

Reply:
That's a fair point. I'll make the change.